codeful-logicapp-sample/test-app/obj/Debug/net8.0/test-app.GeneratedMSBuildEditorConfig.editorconfig

```diff
@@ -0,0 +1,21 @@
+is_global = true
+build_property.TargetFramework = net8.0
+build_property.TargetFramework = net8.0
```

Comment:
lets remove all the artifacts that will be generated at build time e.g. Debug folder